### PR TITLE
Update audit.md

### PR DESCRIPTION
**Pull request by SebastianUA**

### CODE REVIEW

**Comment by sftim** [comment]
Hi. Why should we make this change?

**Comment by SebastianUA** [comment]
> Hi. Why should we make this change?
Hi @sftim 
Because backslash is wrong here! From time to time when I'm copying/pasting those params I must do one more action - to delete `\` from the configuration file of `kube-apiserver.yaml`.

Thanks.

File: content/en/docs/tasks/debug/debug-cluster/audit.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ You can configure the log audit backend using the following `kube-apiserver` fla
 If your cluster's control plane runs the kube-apiserver as a Pod, remember to mount the `hostPath`
 to the location of the policy file and log file, so that audit records are persisted. For example:
 ```shell
-    --audit-policy-file=/etc/kubernetes/audit-policy.yaml \
+    --audit-policy-file=/etc/kubernetes/audit-policy.yaml
```
Comment (sftim) [inline content/en/docs/tasks/debug/debug-cluster/audit.md:149]: In shell (which this is), the trailing backslash is required for continuation.

> A non-quoted backslash (\) is the escape character. It preserves the literal value of the next character that follows, with the exception of <newline>. If a \<newline> pair appears, and the backslash is not itself quoted, the \<newline> is treated as a line continuation (that is, it is removed from the input stream and effectively ignored).

(bash documentation)

In YAML, using a backslash can work like that too. For example:
```yaml
key: "command \
  --foo \
  --bar"
```
which is equivalent to:
```yaml
{
  "key": "command --foo --bar"
}
```

If you want JSON, you will need to adapt the line. That's legit.

Using shell wrapping communicates that it's all one command, and people can easily delete the backslash if they want to put the command on one line.

Comment (SebastianUA) [reply content/en/docs/tasks/debug/debug-cluster/audit.md:149]: Hey @sftim 

If talk about `shell` - yes, you're true. I'm talking about the YAML configuration file: 
```controlplane $ cat /etc/kubernetes/manifests/kube-apiserver.yaml | grep -i "command:" -A10
  - command:
    - kube-apiserver
    - --advertise-address=172.30.1.2
    - --allow-privileged=true
    - --authorization-mode=Node,RBAC
    - --audit-policy-file=/etc/kubernetes/audit-policy.yaml
    - --audit-log-path=/var/log/kubernetes/audit/audit.log
    - --client-ca-file=/etc/kubernetes/pki/ca.crt
    - --enable-admission-plugins=NodeRestriction
    - --enable-bootstrap-token-auth=true
    - --etcd-cafile=/etc/kubernetes/pki/etcd/ca.crt
controlplane $ ```

So, to add auditing there, I must use the off documentation https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/ where I should copy/past some parameters.

Comment (tengqm) [reply content/en/docs/tasks/debug/debug-cluster/audit.md:149]: Okay, This one is not Shell. It is YAML, from kube-apiserver.yaml.
line 148 is misleading.

Comment (sftim) [reply content/en/docs/tasks/debug/debug-cluster/audit.md:150]: If this is an excerpt from YAML, it's **not** a valid YAML list (there should also be a `-` preceding each entry).

If making such a change we should also amend the text to make it clear that we are providing an excerpt from a manifest for a static Pod, as the current page doesn't imply or state that.

Comment (tengqm) [reply content/en/docs/tasks/debug/debug-cluster/audit.md:150]: right，the leading `-`s are missing.

**Comment by SebastianUA** [comment]
Hi @sftim 

@tengqm updated the PR and we can merge it if so.

**Comment by divya-mohan0209** [comment]
Hello @SebastianUA , please can you push an empty commit to retrigger the build?

**Comment by sftim** [comment]
(even better: rebase this PR against _main_)

**Pull request merged**